src/towncrier/test/test_build.py

```diff
@@ -671,3 +671,58 @@ def test_bullet_points_false(self):
             """
             ).lstrip(),
         )
+
+    def test_start_string(self):
+        """
+        Rendered content added after specified start string.
+        """
+        runner = CliRunner()
+
+        with runner.isolated_filesystem():
+            with open("pyproject.toml", "w") as f:
+                f.write(dedent("""\
+                    [tool.towncrier]
+                    start_string="a different start line"
+                """))
+            os.mkdir("newsfragments")
+            with open("newsfragments/123.feature", "w") as f:
+                f.write("Adds levitation")
```

Comment:
I see this pattern many times

I think it can be extracted into a helper ... and the helper can also auto call os.mkdir if not already exists.

```suggestion
            self.createFragment('123.feature', "Adds levitation")
```